packit_service/worker/allowlist.py

```diff
@@ -0,0 +1,346 @@
+        """
+        Check if the user is a packager, by checking if their GitHub
+        username is in the 'packager' group in FAS. Works only the user's
+        username is the same in GitHub and FAS.
+        :param account_login: str, Github username
+        :return: bool
+        """
+
+        try:
+            person = self._fas.person_by_username(account_login)
+        except AuthError as e:
+            logger.error(f"FAS authentication failed: {e!r}")
+            return False
+        except FedoraServiceError as e:
+            logger.error(f"FAS query failed: {e!r}")
+            return False
+
+        if not person:
+            logger.info(f"Not a FAS username {account_login!r}.")
+            return False
+
+        for membership in person.get("memberships", []):
+            if membership.get("name") == "cla_fpca":
+                logger.info(f"User {account_login!r} signed FPCA!")
+                return True
+
+        logger.info(f"Cannot verify whether {account_login!r} signed FPCA.")
+        return False
+
+    def add_account(self, account_login: str, sender_login: str) -> bool:
+        """
+        Add account to allowlist.
+        Status is set to 'waiting' or to 'approved_automatically'
+        if the account is a packager in Fedora.
+        :param sender_login: login of the user who installed the app into 'account'
+        :param account_login: login of the account into which the app was installed
+        :return: was the account (auto/already)-allowlisted?
+        """
+        # TODO: Switch to AllowlistModel
+        if AllowlistModel.get_account(account_login):
+            return True
+
+        # TODO: Switch to AllowlistStatus
+        AllowlistModel.add_account(account_login, AllowlistStatus.waiting.value)
+
+        if self._signed_fpca(sender_login):
+            AllowlistModel.add_account(
+                account_login, AllowlistStatus.approved_automatically.value
+            )
+            return True
+
+        return False
+
+    @staticmethod
+    def approve_account(account_name: str):
+        """
+        Approve user manually
+        :param account_name: account name for approval
+        """
+        AllowlistModel.add_account(
+            account_name=account_name, status=AllowlistStatus.approved_manually.value
+        )
+
+        logger.info(f"Account {account_name!r} approved successfully.")
+
+    @staticmethod
+    def is_approved(account_name: str) -> bool:
+        """
+        Check if user is approved in the allowlist
+        :param account_name: account name to check
+        :return:
+        """
+        account = AllowlistModel.get_account(account_name)
+        if account:
+            db_status = account.status
+            s = AllowlistStatus(db_status)
+            return (
+                s == AllowlistStatus.approved_automatically
+                or s == AllowlistStatus.approved_manually
+            )
```

Comment:
```suggestion
            return AllowlistStatus(db_status) in (
                AllowlistStatus.approved_automatically,
                AllowlistStatus.approved_manually
            )
```

Reply:
thx for bringing it up, i noticed it, must have carried on with renaming and then forgot :)